tests/utils.py

```diff
@@ -62,3 +64,17 @@ def tmp(path=None, teardown=True):
         except OSError as oserr:
             logging.debug("tmp:rmtree failed %s (%s)" % (fname, oserr))
             shutil.rmtree(fname, ignore_errors=True)
+
+
+class ErtTest(ExtendedTestCase):
+    SOURCE_ROOT = SOURCE_DIR
```

Comment:
You can just use SOURCE_DIR directly

Reply:
my mistake will be corrected, I only did it as a hypothetical, but didn't mean to commit it. this should not be done or discussed in this PR. the scope of this issue is to remove __init__ not modify/refactor the content/behavior of the tests.
It will be removed

Reply:
I looked as a fine change. I don't even understand the existing code. How do we get the cmake dir when we are not even running cmake?

Reply:
I'm not aware either, I will follow up with a discussion and issue for this change